sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.institutions.controller.ts

```diff
@@ -73,9 +80,11 @@ const INVALID_TUITION_REMITTANCE_AMOUNT_MESSAGE =
   "Tuition amount provided should be lesser than both (Actual tuition + Program related costs) and (Canada grants + Canada Loan + BC Loan).";
 
 @AllowAuthorizedParty(AuthorizedParties.institution)
-@Controller("institution/location")
-@ApiTags("institution")
-export class ConfirmationOfEnrollmentController extends BaseController {
+@Controller("location")
+@ApiTags(
+  `${ClientTypeBaseRoute.Institution}-location[Confirmation Of Enrollment - CoE]`,
```

Comment:
👍

Reply:
Thanks @ann-aot. I was not so sure about it but I believe that it looks better on Swagger.

Reply:
ya yes